FirebaseFunctions/Sources/Functions.swift

```diff
@@ -39,7 +39,7 @@ internal enum FunctionsConstants {
 /**
  * `Functions` is the client for Cloud Functions for a Firebase project.
  */
-@objc(FIRFunctions) public class Functions: NSObject {
+@objc(FIRFunctions) open class Functions: NSObject {
```

Comment:
We should do the same for all of the classes, right?

Reply:
Done for all public classes now.